Sources/MongoSwift/BSON/Overwritable.swift

```diff
@@ -1,4 +1,5 @@
 import bson
+import Foundation
```

Comment:
Is this required due to the changes below, or was it just something that was missing?

Reply:
Yes, it is required due to the changes below. It is because we are referencing the `Date` type, which requires an `Foundation`.